docs/ca/dev/monitoring/spryker-monitoring-integration/opentelemetry-instrumentation.md

```diff
@@ -11,7 +11,7 @@ This document describes how to configure and instrument application for OpenTele
 
 ## Convention
 
-The current implementation follows [OpenTelemetry Semantic Conventions 1.30.0](https://opentelemetry.io/docs/specs/semconv/).
+The current implementation follows [OpenTelemetry Semantic Conventions 1.36.0](https://opentelemetry.io/docs/specs/semconv/).
```

Comment:
please use this link https://github.com/open-telemetry/semantic-conventions/blob/v1.36.0/docs/README.md
The link that we used previously always lead to the latest version of conventions, that can be misleading